src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js

```diff
@@ -79,7 +61,7 @@ class AccessGroupDynamicPolicyForm extends React.Component {
               content:
                 "How many hours authenticated users can work before refresh",
             }}
-            formName="expiration"
+            formName="dynamic_policies"
```

Comment:
form name should have `this.props.data.name` as par

Reply:
can you explain this? just change to
`formName="this.props.data.name"` ?

Reply:
`formName={this.props.data.name + "-dynamic-policies"}`